src/commands/calculator.ts

```
@@ -1,66 +1,90 @@
       ["7", "8", "9", "*"],
       ["4", "5", "6", "-"],
       ["1", "2", "3", "+"],
-      [".", "0", "00", "="]
+      [".", "0", "00", "="],
     ];
 
-    const components = buttons.map(row => ({
+    const components = buttons.map((row) => ({
       type: 1,
-      components: row.map(label => this.createButton(label))
+      components: row.map((label) => this.createButton(label)),
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.

(lint/complexity/noThisInStatic)